recipe/meta.yaml

```diff
@@ -14,33 +14,60 @@ source:
   sha256: 261663db21bcc1cc232b07ea683252ee6992982276536924271535875f5b0556
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win and vc<14]
   run_exports:
     - {{ pin_subpackage('epoxy', max_pin='x.x') }}
 
 requirements:
   build:
+    - {{ compiler('c') }}
+    # The following CDTs are needed for building against egl/gl.
+    # We only actually need the headers, as epoxy doesn't link to the gl libs.
+    # We can ignore pkg-config saying it can't find egl/gl (due to missing deps
+    # included in the .pc files) since this is only currently used to add
+    # egl/gl as Requires.private dependencies in the epoxy.pc file, which
+    # itself is only used to propagate build flags from the GL libs to epoxy,
+    # and there is nothing interesting to propagate from mesa. If needed, this
+    # should get pkg-config to find egl/gl (check their .pc files to verify,
+    # probably only needed for mesa < 19):
+    #   build:
+    #     - {{ cdt('libxxf86vm-devel') }}   # [linux]
+    #   host:
+    #     - libdrm                 # [linux]
+    #     - xorg-libxdamage        # [linux]
+    #     - xorg-libxext           # [linux]
+    #     - xorg-libxfixes         # [linux]
+    #     - xorg-xf86vidmodeproto  # [linux]
+    # Since these dependencies would be needed wherever epoxy.pc is consumed,
+    # it is simpler to leave them off for now while we can.
+    - {{ cdt('mesa-libegl-devel') }}  # [linux]
+    - {{ cdt('mesa-libgl-devel') }}   # [linux]
     - meson
     # see https://github.com/mesonbuild/meson/issues/7611
     - meson !=0.56.*  # [win]
```

Comment:
Since meson is now at 0.57.2, it is probably safe to remove the constraint for windows.